### PR TITLE
Fixed Chunker Evaluation Iss222 and ChunkAnnotator Init Bug Iss221

### PR DESCRIPTION
**Pull request by qiangning**
It's indeed a git revert thing. I have cleaned things up. Please have a check.
#222 
#221

### CODE REVIEW

File: chunker/scripts/runBenchmarkTest.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ for JAR in `ls $LIB`; do
 done
 
 
-TESTDATA="src/main/resources/test.txt"
+TESTDATA="/shared/corpora/corporaWeb/written/eng/chunking/conll2000distributions/test.txt"
```
Comment (danyaljj) [inline chunker/scripts/runBenchmarkTest.sh]: this change is intentional?

**Comment by danyaljj** [comment]
I think the tests are failing since your branch `qiangning:master` is way behind `IllinoisCogComp:master`.

File: chunker/scripts/runBenchmarkTest.sh (hunk 1 above)
Comment (qiangning) [inline chunker/scripts/runBenchmarkTest.sh]: Yes. But runBenchmarkTest.sh is not used anyhow, you can either adopt it or reject it.

Comment (danyaljj) [inline chunker/scripts/runBenchmarkTest.sh]: What do you mean by "not used anyhow"? 
Why don't we use only the links to [the files that we already have in the repo](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/tree/master/chunker/src/test/resources)?

**Comment by qiangning** [comment]
In the beginning I synced my fork with `upstream`, but the resulting package cannot be compiled (so I later reverted the sync). Are you sure that `IllinoisCogComp:master` is good to run? If yes, let me try to redo things tomorrow.

**Comment by danyaljj** [comment]
> Are you sure that `IllinoisCogComp:master` is good to run? 

Do you see the big green badge on the main page of the repo? 

screen shot 2016-10-02 at 8 25 15 pm

> If yes, let me try to redo things tomorrow.

Try a couple of times; if you fail, bring it to me. Git can be weird, but it is important to master. These are the best occasions to learn it.

**Comment by qiangning** [comment]
Thanks for explaining to me. BTW, what do the other two logos saying "build failing" mean? Are they because of my failed pull request?

**Comment by danyaljj** [comment]
Nothing for now. They are not stable, yet.

File: chunker/scripts/runBenchmarkTest.sh (hunk 1 above)
Comment (qiangning) [inline chunker/scripts/runBenchmarkTest.sh]: This script is running [ChunkTester](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/master/chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkTester.java), which doesn't take any file as input. I think one day someone modified the interface of ChunkTester but didn't update runBenchmarkTest.sh.

Comment (danyaljj) [inline chunker/scripts/runBenchmarkTest.sh]: Ah I see what you saying. Then maybe we can drop `TESTDATA`?

Comment (qiangning) [inline chunker/scripts/runBenchmarkTest.sh]: Yeah. The reason I'm not worried about it is that I think some day we will add a new ChunkerTester which takes a test file name as input; after all, we want more people using our code.

Comment (danyaljj) [inline chunker/scripts/runBenchmarkTest.sh]: maybe add a TODO comment to prevent future confusions?

Comment (qiangning) [inline chunker/scripts/runBenchmarkTest.sh]: Good idea.

**Pull request closed without merging**